src/argilla/client/feedback/dataset/remote.py

```diff
@@ -0,0 +1,257 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import warnings
+from typing import TYPE_CHECKING, Any, Dict, Iterator, List, Optional, Union
+
+from tqdm import trange
+
+from argilla.client.feedback.constants import FETCHING_BATCH_SIZE, PUSHING_BATCH_SIZE
+from argilla.client.feedback.dataset.base import FeedbackDatasetBase
+from argilla.client.feedback.schemas.records import FeedbackRecord, RemoteFeedbackRecord
+from argilla.client.sdk.v1.datasets import api as datasets_api_v1
+
+if TYPE_CHECKING:
+    from uuid import UUID
+
+    import httpx
+
+    from argilla.client.feedback.dataset.local import FeedbackDataset
+    from argilla.client.feedback.types import AllowedFieldTypes, AllowedQuestionTypes
+    from argilla.client.sdk.v1.datasets.models import FeedbackItemModel
+    from argilla.client.workspaces import Workspace
+
+
+warnings.simplefilter("always", DeprecationWarning)
+
+
+class RemoteFeedbackRecords:
+    def __init__(self, client: "httpx.Client", id: "UUID", questions: List["AllowedQuestionTypes"]) -> None:
+        self.client = client
+        self.id = id
```

Comment:
If the id refers to the dataset id, I would say to explicitly call `dataset_id`. Otherwise can be hard to know which id is referring to.

```suggestion
    def __init__(self, client: "httpx.Client", dataset_id: "UUID", questions: List["AllowedQuestionTypes"]) -> None:
        self.client = client
        self.dataset_id = dataset_id
```

Reply:
I agree, renamed that and made it private already!